dependency_support/com_github_openroad/bundled.BUILD.bazel

```diff
@@ -0,0 +1,1732 @@
+    ],
+    alwayslink = True,
+)
+
+genrule(
+    name = "post9",
+    srcs = [
+        "src/stt/src/flt/etc/MakeDatVar.tcl",
+        "src/stt/src/flt/etc/POST9.dat",
+    ],
+    outs = ["POST9.cpp"],
+    cmd = "$(location @tk_tcl//:tclsh) $(location src/stt/src/flt/etc/MakeDatVar.tcl) post9 \"$@\" $(location src/stt/src/flt/etc/POST9.dat)",
+    exec_tools = ["@tk_tcl//:tclsh"],
+)
+
+genrule(
+    name = "powv9",
+    srcs = [
+        "src/stt/src/flt/etc/MakeDatVar.tcl",
+        "src/stt/src/flt/etc/POWV9.dat",
+    ],
+    outs = ["POWV9.cpp"],
+    cmd = "$(location @tk_tcl//:tclsh) $(location src/stt/src/flt/etc/MakeDatVar.tcl) powv9 \"$@\" $(location src/stt/src/flt/etc/POWV9.dat)",
+    exec_tools = ["@tk_tcl//:tclsh"],
+)
+
+cc_library(
+    name = "flute3",
+    srcs = [
+        "src/stt/src/flt/flute.cpp",
+        ":POST9.cpp",
+        ":POWV9.cpp",
+    ],
+    hdrs = [
+        "src/stt/include/stt/flute.h",
+    ],
+    features = ["-use_header_modules"],
+    includes = [
+        "src/stt/include",
+        "src/stt/include/stt",
+    ],
+    deps = [
+        ":logger",
+    ],
+)
+
+cc_library(
+    name = "logger",
+    srcs = [
+        "src/utl/src/Logger.cpp",
+    ],
+    hdrs = [
+        "src/utl/include/utl/Logger.h",
+    ],
+    copts = [
+        "-Wno-error",
+        "-Wall",
+        "-Wextra",
+        "-pedantic",
+        "-Wcast-qual",
+        "-Wredundant-decls",
+        "-Wformat-security",
+    ],
+    includes = [
+        "src/utl/include",
+        "src/utl/include/utl",
+    ],
+    visibility = ["@com_github_openroad//:__subpackages__"],
+    deps = [
+        "@com_github_gabime_spdlog//:spdlog",
+    ],
+)
+
+cc_library(
+    name = "ParquetFP",
+    srcs = glob([
+        "src/mpl/src/ParquetFP/src/*.cxx",
+    ]),
+    hdrs = glob([
+        "src/mpl/src/ParquetFP/src/*.h",
+        "src/mpl/src/ParquetFP/src/Ctainers/*.h",
+    ]),
+    copts = [
+        "-fexceptions",
+        "-Wno-error",
+        "-Wall",
+        "-Wextra",
+        "-pedantic",
+        "-Wcast-qual",
+        "-Wredundant-decls",
+        "-Wformat-security",
+    ],
+    features = ["-use_header_modules"],
+    includes = [
+        "src/mpl/src/ParquetFP/src",
+    ],
+    visibility = ["@com_github_openroad//:__subpackages__"],
+)
+
+cc_library(
+    name = "munkres",
+    srcs = glob([
+        "src/ppl/src/munkres/src/*.cpp",
+    ]),
+    hdrs = glob([
+        "src/ppl/src/munkres/src/*.h",
+    ]),
+    includes = [
+        "src/ppl/src/munkres/src",
+    ],
+    visibility = ["@com_github_openroad//:__subpackages__"],
+)
+
+cc_library(
+    name = "openroad_version",
+    hdrs = [
+        ":OpenRoadVersion",
+    ],
+    includes = [
+        "include/ord",
+    ],
+)
+
+genrule(
+    name = "OpenRoadVersion",
+    srcs = [],
+    outs = ["include/ord/Version.hh"],
+    cmd = """echo -e '
+#define OPENROAD_VERSION "1.0.0-Google"
+#define OPENROAD_GIT_SHA1 "763cc3aa7d4947b496508b6f53054561fa00cc05"
```

Comment:
I think there's a large risk that this will get out of date. Can you replace this with something like N/A or HEAD or something like that? If not, then maybe create constants defined at the top of the file to make it more visible for future BUILD file maintainers?